modules/021-cni-cilium/openapi/config-values.yaml

```diff
@@ -63,6 +63,11 @@ properties:
     default: false
     description: |
       Do not enforce any Network Policy. Just log allowed/disallowed connections.
+  cniExclusive:
```

Comment:
The name for the option is not the best. By reading this it is unclear what it actually does. 

The only argument to support the choice I can find is that the name is the same a in cilium.

Reply:
It is. How about `exclusiveCNIPlugin` and the clear description? (i just committed).